amm/interp/src/main/scala/ammonite/interp/script/AmmoniteBuildServer.scala

```diff
@@ -126,6 +126,11 @@ class AmmoniteBuildServer(
     clientOpt = Some(client)
   }
 
+  // Stubs
+  override def buildTargetDependencyModules(dmp: DependencyModulesParams): CompletableFuture[DependencyModulesResult] = ???
```

Comment:
I'd rather add those to [`DummyBuildServerImplems`](https://github.com/com-lihaoyi/Ammonite/blob/e445eb40cee0b6548852fb521e48795c292f029f/amm/interp/src/main/scala/ammonite/interp/script/DummyBuildServerImplems.scala), and return empty results rather than throwing when possible.

Reply:
Adjusting the stubs to use DummyBuildServerImplems